xmtp/src/conversations.rs

```diff
@@ -457,6 +619,10 @@ mod tests {
         ClientBuilder, Fetch,
     };
 
+    fn init() {
+        let _ = env_logger::builder().is_test(true).try_init();
+    }
```

Comment:
This allows conditional logging inside of tests. If called from a test: All Errors are logged, and log entries are included if the test fails

Reply:
Ooh. That's very helpful. I've been replacing our log statements with `println`s when debugging tests, and then switching them back.